### PR TITLE
Switch Key Vault canary region to Central US

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Comment by heaths** [comment]
Talking with @benbp offline, we'll need this change anyway since the linux App Service is only available in centraluseuap anyway.

**Comment by benbp** [comment]
@heaths my onboarding ticket is still pending.

**Review by christothes — APPROVED** [review]

File: sdk/keyvault/tests.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ extends:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
       Canary:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-        Location: 'eastus2euap'
+        Location: 'centraluseuap'
```
Comment (pallavit) [inline sdk/keyvault/tests.yml:15]: Do we have to be in canary? Was this mostly to test early? Canary tends to always be less stable than the production environment so was curious.  Also is this a .NET decision or something that we do across? I see different SDKs doing it differently.

Comment (heaths) [reply sdk/keyvault/tests.yml:15]: Canary was an intentional decision, yes. We provide backstop testing so it helps spot any regressions early.

Comment (pallavit) [reply sdk/keyvault/tests.yml:15]: got it, thanks.

**Comment by heaths** [comment]
The problems with China are all known timeouts that started happening about a month ago. Tracking resolution in https://github.com/Azure/azure-sdk-tools/issues/6175. Will override.

**Pull request merged**